packages/consumption/src/modules/requests/itemProcessors/createAttribute/CreateAttributeRequestItemProcessor.ts

```diff
@@ -20,28 +20,44 @@ export class CreateAttributeRequestItemProcessor extends GenericRequestItemProce
         if (requestItem.attribute instanceof IdentityAttribute) {
             if (senderIsAttributeOwner) {
                 return ValidationResult.error(
-                    CoreErrors.requests.invalidRequestItem("Cannot create own Attributes with a CreateAttributeRequestItem. Use a ShareAttributeRequestItem instead.")
+                    CoreErrors.requests.invalidRequestItem("Cannot create own IdentityAttributes with a CreateAttributeRequestItem. Use a ShareAttributeRequestItem instead.")
                 );
             }
 
-            if (recipientIsAttributeOwner || ownerIsEmptyString || typeof recipient === "undefined") {
+            if (recipientIsAttributeOwner || ownerIsEmptyString) {
                 return ValidationResult.success();
             }
 
+            if (recipient !== undefined) {
+                return ValidationResult.error(
+                    CoreErrors.requests.invalidRequestItem(
+                        "The owner of the provided IdentityAttribute for the `attribute` property can only be the Recipient's Address or an empty string. The latter will default to the Recipient's Address."
+                    )
+                );
+            }
+
             return ValidationResult.error(
                 CoreErrors.requests.invalidRequestItem(
-                    "The owner of the given `attribute` can only be the recipient's address or an empty string. The latter will default to the recipient's address."
+                    "The owner of the provided IdentityAttribute for the `attribute` property can only be an empty string. It will default to the Recipient's Address."
```

Comment:
why is an undefined recipient now an error? also the error message then shouldn't reference a recipient

Reply:
The Recipient is an optional parameter of the function. It is known and is passed when the Request is sent via a Message. It is not known and is not passed if the Request is sent via a RelationshipTemplate. Accordingly, a case distinction is made, as the Sender of the Request has different valid input options for the `owner` of the Attribute depending on whether the Request is sent via Message or RelationshipTemplate.